prog/postgres/postgres_server_nexus.rb

```diff
@@ -279,6 +280,69 @@ def before_run
     COMMAND
 
     vm.sshable.cmd(command)
+    hop_configure_pgbouncer
+  end
+
+  label def configure_pgbouncer
+    pgbouncer_password = SecureRandom.urlsafe_base64(15)
+
+    pgbouncer_setup_query = <<~PGBOUNCER_SETUP
+      BEGIN;
```

Comment:
I think it would be better to move this to `initialize-empty-database` rhizome script